include/zephyr/drivers/video.h

```diff
@@ -1651,32 +1639,68 @@ static inline unsigned int video_bits_per_pixel(uint32_t pixfmt)
  */
 
 /**
- * @name MIPI CSI2 Data-types
+ * @name MIPI CSI-2 Data Types
+ * @brief Standard MIPI CSI-2 data type identifiers for camera sensor interfaces
+ *
+ * These constants define the data type field values used in MIPI CSI-2 packet headers to identify
+ * the format and encoding of transmitted image data. The data type field is 6 bits wide, allowing
+ * values from 0x00 to 0x3F.
  *
  * @{
  */
-#define VIDEO_MIPI_CSI2_DT_NULL	0x10
-#define VIDEO_MIPI_CSI2_DT_BLANKING	0x11
-#define VIDEO_MIPI_CSI2_DT_EMBEDDED_8	0x12
-#define VIDEO_MIPI_CSI2_DT_YUV420_8	0x18
-#define VIDEO_MIPI_CSI2_DT_YUV420_10	0x19
+
+/** NULL data type - used for padding or synchronization */
+#define VIDEO_MIPI_CSI2_DT_NULL                 0x10
+/** Blanking data - horizontal/vertical blanking information */
+#define VIDEO_MIPI_CSI2_DT_BLANKING             0x11
+/** Embedded 8-bit data - sensor metadata or configuration data */
+#define VIDEO_MIPI_CSI2_DT_EMBEDDED_8           0x12
+/** YUV 4:2:0 format with 8 bits per component */
+#define VIDEO_MIPI_CSI2_DT_YUV420_8             0x18
+/** YUV 4:2:0 format with 10 bits per component */
+#define VIDEO_MIPI_CSI2_DT_YUV420_10            0x19
+/** YUV 4:2:0 CSPS (Chroma Sub-sampled Packed Stream) 8-bit format */
 #define VIDEO_MIPI_CSI2_DT_YUV420_CSPS_8	0x1c
+/** YUV 4:2:0 CSPS (Chroma Sub-sampled Packed Stream) 10-bit format */
 #define VIDEO_MIPI_CSI2_DT_YUV420_CSPS_10	0x1d
```

Comment:
I had to check what CSPS mean ^_^'

The [MIPI draft](https://github.com/DRubioG/MIPI_Specification/blob/main/_MIPI%20Alliance%20Specification%20for%20Camera%20Serial%20Interface%202%20(CSI-2).pdf) says "Chroma Shifted Pixel Sampling"

It refers to where the colors (in grey in the diagram below, 4 times lower resolution than grayscale in YUV 4:2:0) are picked according to the grayscale values (in white in the diagram below).

![Image](https://github.com/user-attachments/assets/d13ce7c3-c266-45c2-adab-c0165ebf798b)

https://en.wikipedia.org/wiki/Chroma_subsampling#Sampling_positions

```suggestion
/** YUV 4:2:0 CSPS (Chroma-Shifted Pixel Sampling) 8-bit format */
#define VIDEO_MIPI_CSI2_DT_YUV420_CSPS_8	0x1c
/** YUV 4:2:0 CSPS (Chroma-Shifted Pixel Sampling) 10-bit format */
#define VIDEO_MIPI_CSI2_DT_YUV420_CSPS_10	0x1d
```